### DISCUSSION TITLE
layouts to do

### DISCUSSION THREAD

**Discussion by bangbangyouall** [lightbeam24/CustomTube#81] [Ideas]
stargazers 1st

idk any other 2009s

more layouts for aozora to make it an actual category

![image](https://github.com/lightbeam24/CustomTube/assets/109046841/27f7dd55-6efc-4243-ae2e-32fee3d42bf9)
modify 2011 to do this, 2010

same thing for cospan

![image](https://github.com/lightbeam24/CustomTube/assets/109046841/09cb3664-6378-46d3-a11f-e328fc77771b)
if it's possible, early 2012, cosmic panda beta
![image](https://github.com/lightbeam24/CustomTube/assets/109046841/54591d14-fe49-4bb1-a0c3-08092b9e76d8)
put the 2013 watchpage type onto 2012, mid-2012, cosmic panda initial release before redesign

new hitchhiker

![image](https://github.com/lightbeam24/CustomTube/assets/109046841/dd0b8fd3-0a12-4d90-ab40-5d63be5dfd76)
just change the top bar of 2014 like this, early 2014, hitchhiker beta layout

a new polymer layout

![image](https://github.com/lightbeam24/CustomTube/assets/109046841/19a75eb8-1913-40a5-bbdf-af57cbe7ebd1)
![image](https://github.com/lightbeam24/CustomTube/assets/109046841/bfd8f0ad-1550-4312-9486-9ca962194838)
modify 2019 like this, 2018, i didn't have any references so here have this accurate userstyle

**Comment by lightbeam24** [comment]
I don't plan to add any more layouts for a while yet now, but this is a good list for when I do get back to adding them, so thanks.

The only layouts I'm certain I'm going to add eventually are:
Early 2009
Mid 2009
2010

> same thing for cospan

I don't know about Mid 2011 Cosmic Panda beta, it might not be possible to get it accurate enough to where it's worth adding.
 
With Cosmic Panda I was actually planning on doing something different. Cosmic Panda had 3 revisions throughout 2012. Eventually the current 2012 layout will become the Late 2012 Layout, and then Early and Mid 2012 will be added. The 2012 layout is too inaccurate to justify this yet though.

> if it's possible, early 2012, cosmic panda beta

That's actually a scrapped layout called Epic Panda. I wasn't originally planning on adding it, but maybe, we'll see.

> just change the top bar of 2014 like this, early 2014, hitchhiker beta layout

We'll see.

> a new polymer layout

Which layout is this supposed to be? It just looks like 2019 to me.

**Comment by bangbangyouall** [comment]
"I don't plan to add any more layouts for a while yet now, but this is a good list for when I do get back to adding them, so thanks."

my pleasure

"I don't know about Mid 2011 Cosmic Panda beta, it might not be possible to get it accurate enough to where it's worth adding."

i was talking about how the cosmic panda category needs more like how aozora does

"With Cosmic Panda I was actually planning on doing something different. Cosmic Panda had 3 revisions throughout 2012. Eventually the current 2012 layout will become the Late 2012 Layout, and then Early and Mid 2012 will be added. The 2012 layout is too inaccurate to justify this yet though."

i wanna see what the early & mid looks like, very interesting

"That's actually a scrapped layout called Epic Panda. I wasn't originally planning on adding it, but maybe, we'll see."

i hope it's possible

"We'll see."

i hope it ends up happening, love to see 2014 history get some recognition

"Which layout is this supposed to be? It just looks like 2019 to me."

it's slightly different, called 2018

**Comment by lightbeam24** [comment]
> i wanna see what the early & mid looks like, very interesting

It's mostly the homepage that's different between them.

Early 2012: https://web.archive.org/web/20120201001734/http://www.youtube.com/
Mid 2012: https://web.archive.org/web/20120531234321/http://www.youtube.com/
Late 2012: https://web.archive.org/web/20120811021208/http://www.youtube.com/

> i hope it's possible

It should be. I'd just need to find more screenshots (I think I recall seeing some on a forum somewhere)

> it's slightly different, called 2018

We'll see. As it is, the 2019 layout option is actually more of a mix of 2017-2019 polymer from whatever archives and videos I could find, rather than an actual accurate 2019 recreation.
